src/main/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/Database.java

```diff
@@ -53,15 +55,16 @@ public RangeFilter create(RangeFilter filter) throws InvalidFilterTypeException,
                     statement.setLong(5, filter.getDbMaxLong());
                 }
                 case TIME -> {
+                    Calendar cal = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
                     statement = connection.prepareStatement(
                             "INSERT INTO SortingRangeTimeView (name, description, productAttribute, min, max) VALUES (?, ?, ?, ?, ?)",
                             queryAttributes
                     );
                     statement.setString(1, filter.getName());
                     statement.setString(2, filter.getDescription());
                     statement.setString(3, filter.getProductAttribute());
-                    statement.setTimestamp(4, Timestamp.from(filter.getDbMinInstant()));
-                    statement.setTimestamp(5, Timestamp.from(filter.getDbMaxInstant()));
+                    statement.setTimestamp(4, new Timestamp(filter.getDbMinInstant().toEpochMilli()), cal);
+                    statement.setTimestamp(5, new Timestamp(filter.getDbMaxInstant().toEpochMilli()), cal);
```

Comment:
Why change the way we create the timestamp? 
The important thing seems to be passing in the calendar argument.
I tried using our original Timestamp.from(filter.getDbMaxInstant() and changing the timezone on the pc to utc +05:00 and the tests are still passing

Reply:
I just copied what I found on stackoverflow: https://stackoverflow.com/a/6627999
If it works without using epoch, then we can just use that

Reply:
I will push the change and approve